src/index.js

```diff
@@ -69,6 +79,7 @@ class LndEngine {
     this.symbol = symbol
     this.minVersion = minVersion
     this.currencyConfig = currencies.find(({ symbol }) => symbol === this.symbol)
+    this.secondsPerBlock = 600 // can be overridden by config.json setting
```

Comment:
why have this default in here? Seems like it's confusing more than anything

Reply:
It's for the type checker. We could also do a ts-ignore comment but was trying to avoid it.

Reply:
hmm, feels a little bit weird, but I suppose it's ok.